lib/components/Navigation/Breadcrumb/index.tsx

```diff
@@ -0,0 +1,29 @@
+import { Icon, IconType } from "@/components/Utilities/Icon"
+import { ChevronRight } from "@/icons"
+import { forwardRef } from "react"
+
+interface BreadcrumbsType {
+  icon: IconType
+  route: string
+  title: string
+}
+
+export const Breadcrumb = forwardRef<HTMLDivElement, BreadcrumbsType>(
```

Comment:
Have you considered checking out ShadCN's breadcrumb component? https://ui.shadcn.com/docs/components/breadcrumb